index.html

```diff
@@ -1614,13 +1614,54 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
   </p>
   </section> <!-- release NFC -->
 
+  <section data-dfn-for="NDEFCompatibility">
+      <h3>The <dfn>NDEFCompatibility</dfn> enum</h3>
+        <p>
+          To describe what NDEF compatible devices are accepted as
+          vendor specific tags exist that support NDEF but which
+          are not universally supported by all NFC readers, or by
+          the NFC standard.
+        </p>
+        <pre class="idl">
+          enum NDEFCompatibility {
+            "universal",
+            "vendor",
+            "all"
```

Comment:
IMHO ```"all"``` is not suggestive for its stated meaning
```The enum value representing all NDEF compatible devices that the reader chip can read.```
I think "ndef" would be better.

Then, "universal" could become "all-nfc" or just "nfc"?
```The enum value representing all active and passive NFC devices, supported by the NFC standard.```

Reply:
Maybe "any" instead of "all" - that is actually what I meant.

What do you think about "nfc-forum" ?

Reply:
"any" sounds good, in the sense that any device/tag the reader can communicate with. But the text specifies it has to be NDEF based, so it actually means "any NDEF". If we really mean NDEF-only, then IMHO we should use "ndef", otherwise we can use "any".

"nfc-forum" is fine.